lib/gitx/cli/integrate_command.rb

```diff
@@ -9,8 +9,8 @@ module Cli
     class IntegrateCommand < BaseCommand
       include Gitx::Github
       desc 'integrate', 'integrate the current branch into one of the aggregate development branches (default = staging)'
-      method_option :resume, :type => :string, :aliases => '-r', :desc => 'resume merging of feature-branch'
-      method_option :comment, :type => :boolean, :aliases => '-c', :desc => 'add a comment to the pull request for this branch. Creates a new PR if none exists.'
+      method_option :resume, type: :string, aliases: '-r', desc: 'resume merging of feature-branch'
```

Comment:
Line is too long. [99/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.